LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+(C) Copyright 2022 Hewlett Packard Enterprise Development LP
```

Comment:
Maybe a lawyer question but does HPE really have a copyright claim on the MIT license?

Reply:
Already started an email thread to get this clarified, will update as per their suggesion.